modules/admin_manual/pages/maintenance/package_upgrade.adoc

```diff
@@ -1,21 +1,30 @@
 = Upgrade ownCloud From Packages
 :toc: right
+:packages-url: https://owncloud.com/older-versions/#server
 
-== Upgrade Quickstart
+== Upgrade Steps
 
 The alternative to a manual upgrade is configuring your system to use ownCloud’s
 https://download.owncloud.org/download/repositories/stable/owncloud/[Open Build Service] repository.
-Then stay current by using your Linux package manager to install fresh ownCloud packages. 
-After installing upgraded packages you must run a few more steps to complete the upgrade. 
-These are the basic steps to upgrading ownCloud:
+Then stay current by using your Linux package manager to install fresh ownCloud packages. However, you should exclude the ownCloud package during system upgrades. For more information, check out the section on xref:installation/linux_packetmanager_install.adoc[Linux Package Manager Installation]
+
+IMPORTANT: This approach should not be used unattended nor in clustered setups.
+
+In general, we discourage upgrades with a Linux package manager because you might encounter unwanted side effects and you'll have to manage the PHP installation separately. For further information on upgrading PHP, see section xref:upgrading/upgrade_php.adoc[Upgrade PHP on RedHat 7 and CentOS 7]
+
+If you want to proceed anyway, read the xref:release_notes.adoc[the release notes] for important information first.
+
+Before installing upgraded packages, perform the following steps: 
 
 * Disable all xref:maintenance/manual_upgrade.adoc#review-third-party-apps[third-party apps].
 * Make xref:maintenance/backup.adoc[a fresh backup].
-* Upgrade your ownCloud packages.
-* Run xref:configuration/server/occ_command.adoc#command-line-upgrade[occ upgrade]
-+
+
+Now you can upgrade your ownCloud packages, then run xref:configuration/server/occ_command.adoc#command-line-upgrade[{occ-command-example-prefix} upgrade].
+
 NOTE: The optional parameter to skip migration tests was removed in ownCloud 10.0. See xref:maintenance/upgrade.adoc[Testing a Migration] for background information.
 
+After the the upgrade is finished, perform the following actions:
```

Comment:
@EParzefall @mmattel:

`the the` twice here.

Reply:
@hoho thanks for catching

Reply:
Yes, thank you @ho4ho. Fixed in the latest PR #3563.